src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToDefenceSpecCallbackHandler.java

```diff
@@ -268,6 +278,11 @@ private CallbackResponse populateCaseData(CallbackParams callbackParams) {
                                              .build());
         }
 
+        if (V_1.equals(callbackParams.getVersion()) && featureToggleService.isPinInPostEnabled()) {
```

Comment:
After merging the ticket CIV-5369, this will change to V_2

Reply:
ok.